indico/web/templates/forms/location_widget.html

```diff
@@ -1,34 +1,66 @@
+{# The location widget provides 3 core fields to describe the location; venue,
```

Comment:
wouldn't this whole text make more sense in the docstring of `IndicoLocationField`?